### DISCUSSION TITLE
How to re-sign in after I change my Microsoft account passowrd

### DISCUSSION THREAD

**Discussion by GalaxiumNation** [abraunegg/onedrive#2684] [Q&A] [answered]
Long story short I changed my password a few times so I could fix some issues cross platform. Just a few minutes ago I changed it again and I don't know how to log back in to the onedrive sync thing

**Comment by abraunegg** [comment, marked as answer]
@GalaxiumNation 
This is clearly documented in the documentation. Please read:
* https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#handling-a-onedrive-account-password-change
* https://github.com/abraunegg/onedrive/blob/onedrive-v2.5.0-release-candidate-1/docs/usage.md#handling-a-microsoft-onedrive-account-password-change
